### PR TITLE
Fix #4043 - Change localization note and errorLoadingText

### LINKED ISSUE #4043: Copy Source Url" should be changed to "Copy Source URI"
It may be a non-Internet address, like chrome://, file://, resource://, moz-extension:// and more. As well as LOCALIZATION NOTE and "errorLoadingText2=Error loading this URL: %S".

the [property] is defined here

Filing from https://bugzilla.mozilla.org/show_bug.cgi?id=1379293

[property]:https://hg.mozilla.org/mozilla-central/file/default/devtools/client/locales/en-US/debugger.properties#l17

### PR DESCRIPTION
**Pull request by abhinav-koppula**
Associated Issue: #4043 

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes

* change localization note of `copySourceUrl` and `errorLoadingText2`

### CODE REVIEW

File: assets/panel/debugger.properties [hunk 1]
```diff
@@ -20,8 +20,8 @@ copySource=Copy
 copySource.accesskey=y
 
 # LOCALIZATION NOTE (copySourceUrl): This is the text that appears in the
-# context menu to copy the source URL of file open.
-copySourceUrl=Copy Source URL
+# context menu to copy the source URI of file open.
+copySourceUrl=Copy Source URI
```
Comment (flodolo) [inline assets/panel/debugger.properties]: You need to change string IDs (also pay attention to localization note and accesskey)
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings

Comment (jasonLaster) [reply assets/panel/debugger.properties]: lets switch this key to `copySourceURI`

File: assets/panel/debugger.properties [hunk 2]
```diff
@@ -507,7 +507,7 @@ loadingText=Loading\u2026
 
 # LOCALIZATION NOTE (errorLoadingText2): The text that is displayed in the debugger
 # viewer when there is an error loading a file
-errorLoadingText2=Error loading this URL: %S
+errorLoadingText2=Error loading this URI: %S
```
Comment (flodolo) [inline assets/panel/debugger.properties]: To clarify, you need it also in this one (`errorLoadingText3` works.)

**Comment by codehag** [comment]
@abhinav-koppula Hey! thanks for the work, how are things coming along? anything we can do to help?

**Comment by abhinavkoppula** [comment]
@codehag : sorry for the delay, was caught up in other issues. Sure, I'll address the comments and send a new PR again.

**Comment by abhinav-koppula** [comment]
@codehag : Sorry for the delay, was caught up in some issues. Sure, I'll address the comments and send a new PR.

**Comment by codehag** [comment]
@abhinav-koppula no worries! checking in, in case there was something sticky :)

**Comment by jasonLaster** [comment]
@abhinav-koppula looks like a rebase is needed

**Pull request merged**